app/views/orders/new.html.erb

```diff
@@ -12,7 +12,7 @@
       </tr>
     </thead>
     <tbody>
-      <% @order_products.each do |order_product| %>
+      <% @order.order_products.each do |order_product| %>
```

Comment:
Maybe it would be better to refactor association names to have `@order.items` instead of `@order.order_products`?